nerima/src/test/java/brocoeur/example/controller/NerimaControllerTest.java

```diff
@@ -55,6 +54,19 @@ void shouldTestCoinTossStrategy() {
 
         // Then
         var expectedStart = "67890 used strategy: RandomCoinToss and will play: ";
```

Comment:
This is unused, can you remove the line ?

Reply:
fixed